tests/test_solution.py

```diff
@@ -411,21 +410,32 @@ def test_get_amount(s3, s5):
         "mol",
         "eq",
     ]
-    # TODO - make this test more precise i.e. test numerical values
+    # Check that get_amount returns a valid quantity for all units.
     for u in TEST_UNITS:
```

Comment:
Please keep the TODO comment line that was here before. It's fine to keep the additional comment line that you put in, too.